benefits/enrollment/templates/enrollment/index.html

```diff
@@ -1,108 +1,113 @@
 {% extends "core/base.html" %}
+{% load i18n %}
+
+{% translate "core.buttons.wait" as loading_text %}
+
+{% block page_title %}
+  {% translate "enrollment.pages.index.title" %}&nbsp;|&nbsp;
+{% endblock page_title %}
 
 {% block nav-buttons %}
   {% include "core/includes/sign-out-link.html" %}
 {% endblock nav-buttons %}
 
 {% block headline %}
   <div class="col-lg-6">
-    <h1 class="pb-lg-8 pb-4">{{ page.headline }}</h1>
+    <h1 class="pb-lg-8 pb-4">{% translate "enrollment.pages.index.headline" %}</h1>
   </div>
 {% endblock headline %}
 
 {% block inner-content %}
-  <div class="col-12 col-sm-12 col-lg-10">{% include "core/includes/media-list.html" with media=media %}</div>
-
-  {% comment %} This Javascript code must be before the forms. {% endcomment %}
-  {% if payment_processor %}
-    <script nonce="{{ request.csp_nonce }}">
-            var startedEvent = "started payment connection", closedEvent = "closed payment connection";
+  <div class="col-12 col-sm-12 col-lg-10">
+    <ul class="media-list mx-0 px-0 d-flex justify-content-center flex-column">
+      {% include "enrollment/includes/media-item--bankcardcheck--index.html" %}
+    </ul>
+  </div>
+  {% comment %}
+    This Javascript code is partially generated by this template and so it must
+    come before the forms, which are rendered at just before the {% endblock inner-content %}
+  {% endcomment %}
+  {% url "enrollment:token" as access_token_url %}
+  {% translate "core.buttons.wait" as loading_text %}
```

Comment:
I noticed we have this code on line 4 and this line 31. Is there a reason we need both?

Reply:
It turns out we do need both, but the other one was on the wrong line: https://github.com/cal-itp/benefits/pull/1502#issuecomment-1629832107